databases/week1/homework.sql

```diff
@@ -0,0 +1,12 @@
+select * from task
```

Comment:
First of all, it's better practice to use uppercase for writing SQL commands. 
FX: `SELECT * FROM task; `
Next thing: you missed `;` at the end of the string which is required because otherwise you won't get a result of your query.